docs/fx-core/bot-help.md

```diff
@@ -65,4 +65,23 @@ Toolkit v2 will pop-up a dialog to show the target message endpoint for bot. Use
 3. Paste the target message endpoint in text box `Messaging endpoint`.
 ![image](../images/fx-core/bot/update-message-endpoint-azure.png)
 
-After updating message endpoint is done, continue to deploy and try the remote experience.
\ No newline at end of file
+After updating message endpoint is done, continue to deploy and try the remote experience.
+
+## Configuration schema for bot
+This section is to describe configuration items in `fx-resource-bot` section of `.fx/env.default.json`.
+
+Config Name | Config Type | Description
+------|------|------
+wayToRegisterBot|string|The way to register bot, one is `create-new`, the other is `reuse-existing`.
+skuName|string|The sku name for azure web app which is used for hosting the bot project on Azure.
```

Comment:
can we give some examples of skuName value for user

Reply:
sure